cdi-core/src/main/java/com/linkedin/cdi/factory/http/HttpRequestMethod.java

```diff
@@ -55,6 +55,27 @@
  */
 
 public enum HttpRequestMethod {
+  /**
+   * Note: This works when the URI template has all the variable parameters.
+   * If there are any additional parameters, then URIBuilder encodes all parameters while building.
+   */
+  GET_XE("GET_XE") {
+    @Override
+    protected HttpUriRequest getHttpRequestContentJson(String uriTemplate,
+        JsonObject parameters, JsonObject payloads)
+        throws UnsupportedEncodingException {
+      Pair<String, JsonObject> replaced = VariableUtils.replaceWithTracking(uriTemplate, parameters, false);
```

Comment:
May need some more context here. I am assuming that this false flag is internal to DIL and should not be exposed in the user configuration?

Reply:
Yes, it is internal to DIL. 
We make a decision to encode on not depending on http headers/method.